src/online2bin/online2-tcp-nnet3-decode-faster_word_timestamps.cc

```diff
@@ -318,7 +321,14 @@ int main(int argc, char *argv[]) {
               long int timestamp = tp.tv_sec * 1000 + tp.tv_usec / 1000;
               std::string current_block = std::to_string(block);
               std::string current_timestamp = std::to_string(timestamp);
-              current_hypothesis = "{\"block\":" + current_block + ", " + "\"timestamp\": " + current_timestamp + ", " + "\"words\":[" + current_hypothesis + "]}";
+              current_hypothesis = "\"block\":" + current_block + ", " + "\"timestamp\": " + current_timestamp + ", " + "\"words\":[" + current_hypothesis + "]}";
```

Comment:
Can we maybe put this code into a standalone function as the same code is duplicated below starting around line number 400?